### DISCUSSION TITLE
Bugged in deep

### DISCUSSION THREAD

**Discussion by maxlutz857** [community/community#81326] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Thought I got rid of it but tried to log in irs tonight and phone went crazy. My personal Gmail was turned into a workplace account so a unauthorized person could become admin over phone... I think it's roomates

**Comment by maxlutz857** [comment]
I need a new os to download on my sd so I can reset phone and load from sd cuz after factory reset the BUG IS STILL HERE AND HERE ANDHERE ....

**Comment by maxlutz857** [comment]
Thank u.....

**Comment by Akash1134** [comment, marked as answer]
Hi @maxlutz857 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
